hack/bin/bombon.hs

```diff
@@ -1,18 +1,23 @@
-#!/usr/bin/env -S nix -Lv run github:wireapp/ghc-flakr/99fe5a331fdd37d52043f14e5c565ac29a30bcb4
+#!/usr/bin/env -S nix -Lv run github:wireapp/ghc-flakr/6311bb166bf835d4a587fe1661b86c9a1426f212
 {-# LANGUAGE DataKinds #-}
+{-# LANGUAGE LambdaCase #-}
+{-# OPTIONS_GHC -Wall #-}
```

Comment:
```suggestion
{-# OPTIONS_GHC -Wall #-}

-- See https://github.com/nikstur/bombon for more background.
```

it would also be helpful to explain here in one or two lines what bombon.hs and Sbom.hs do, and how they relate to each other.

Reply:
Good idea, thank you. I have written some paragraphs and I guess it makes sense to put them somewhere in the rep.

Reply:
Do you have an idea where I could put a loose markdown file that I could link to? 😅

(I’m guessing the confluence doesn’t make much sense as it isn’t public)

Reply:
There's a docs folder but I'd inline it if possible, or at least add something in-place. If the write up is too long, you can link it here instead.